### PR TITLE
Enable TextFields lint

### LINKED ISSUE #10503: Enable Lint: `TextFields`
We perform lint checking to ensure code correctness and improve performance, but many of the system lint checks have not been enabled.

The goal of this issue is to enable the lint check: `TextFields`

Line: https://github.com/ankidroid/Anki-Android/blob/6260cd6e25f67f331833bbd0981f2cda5a426dde/lint-release.xml#L330

To do this: 

* Change `ignore` to `fatal`
* Move the line in the XML to be somewhere near the top of the file (before the large block of `ignore` elements. 
  * The exact position is up to you, but if there's no comment associated with the check, a large nearly alphabetical block of `fatal` warnings seems sensible
* Run `gradlew lint` in the Android Studio terminal and fix the errors which occur.
  * `@Suppress(str)` (Kotlin) or `@SuppressWarnings(str)`  (Java) the warnings if they are valid. Android Studio's quickfixes menu will offer options to suppress
    * Use your judgment, but aim to suppress as close to the error as possible (line/method) unless the class specifically has a reason why it will violate a lint check many times
  * If it was an error that we missed, fix it.
* Test your changes
* Submit a Pull Request

### PR DESCRIPTION
**Pull request by Prince-kushwaha**
## Purpose / Description
Enable TextFields lint

## Fixes
Fixes #10503

## Approach

1.Changed the severity value of the  TextFields lint from ignore to fatal
2.Fixed the following problems appearing on running gradlew lint
Screenshot 2022-03-12 205052

## Checklist
_Please, go through these checks before submitting the PR._

- [x] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [x] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [x] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

**Review by david-allison — APPROVED** [review]
Looks good to me!

File: lint-release.xml [hunk 1]
```diff
@@ -51,6 +51,7 @@
     <issue id="RtlEnabled" severity="fatal" />
     <issue id="RtlHardcoded" severity="fatal" />
 
+    <issue id="TextFields" severity="fatal" />
```
Comment (ShridharGoel) [inline lint-release.xml:54]: Would be good to have a line break after this.

Comment (Prince-kushwaha) [reply lint-release.xml:54]: ok

**Review by ShridharGoel — APPROVED** [review]

**Comment by ShridharGoel** [comment]
@Prince-kushwaha Can you squash the commits?

**Comment by Prince-kushwaha** [comment]
> @Prince-kushwaha Can you squash the commits?

NO

**Comment by Akshay0701** [comment]
> > @Prince-kushwaha Can you squash the commits?
> 
> NO

you can, if your not familiar with git commands, then you can also use the tools android studio provide to handle this git command(its easy to use), follow the steps mentioned in the [link](https://stackoverflow.com/questions/37122342/squash-git-branch-commits-in-android-studio) to squash the commits,

currently, this pr is in conflict, kindly rebase the current branch with the master branch.

**Pull request merged**